apps/parse/test/parse/calendar_attributes_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Parse.CalendarAttributesTest do
+  @moduledoc false
+  use ExUnit.Case, async: true
+  alias Parse.CalendarAttributes
+
+  test "parses calendar entries" do
+    blob = """
+    service_id,service_description,service_schedule_name,service_schedule_type,service_schedule_typicality
+    BUS12019-hba19ns1-Weekday-02,Weekday schedule (no school),Weekday (no school),Weekday,1
```

Comment:
What happens when typicality is the empty string? What about when the others are the empty string?